obelisk_ws/src/obelisk_ros/config/dummy_cpp.yaml

```diff
@@ -82,6 +82,20 @@ onboard:
           sensor_settings:
           - topic: /obelisk/dummy/sensor
             dt: 0.001
-            sensor_type: jointpos
+            sensor_type: ObkJointEncoders
             sensor_names:
-            - sensor_joint1
+            - joint_pos: jointpos
```

Comment:
for documentation/tutorial purposes, let's leave a comment here explaining how the dict structure works in this field:

the key is the name of the mujoco sensor as specified in the xml. the value is the field which the mujoco sensor should be associated with in the Obelisk message.